### ISSUE TITLE
Correct CSS to match brand color palette and meet GAR requirements

### ISSUE THREAD

**Issue by atsansone** [flutter/website#9427]
Five of fifteen colors in the [_variables.scss](https://github.com/flutter/website/blob/main/src/_sass/base/_variables.scss) match the Flutter branding guidelines. This results in multiple violations of [WCAG 2.0 G18](https://www.w3.org/TR/WCAG20-TECHS/G18.html) and GAR criteria 1.4.

These colors should be updated to the proper color and contrast. 

_buttons.scss
- [ ] Update .btn-primary to #0468D7

_banner.scss
- [ ] Update .site-banner to #0553B1
- [ ] Update .site-banner a to #B8EAFE

_reboot.scss
- [ ] Update a to #0468D7

_code.scss
- [x] Update code to #158477

_rfs.scss
- [x] Update h1 to 3rem
- [x] Update h2 to 2rem
- [x] Update h3 to 1.5rem
- [x] Update h4 to 1.3125rem
- [x] Update h5 to 1.125rem

**Comment by parlough** [comment]
![Photo of a gar underwater](https://github.com/flutter/website/assets/18372958/c76d7f5a-e21e-4941-81df-f51623a6d9ee)

**Comment by parlough** [comment]
As a note, don't worry about the code snippet colors as part of this work. Other work I'm doing (https://github.com/dart-lang/site-www/issues/3846) and hopefully opening a PR for soon (first on site-www) will completely overhaul those.

**Comment by atsansone** [comment]
This removed 68,838 errors.
